src/test/java/de/tum/in/www1/artemis/plagiarism/PlagiarismCaseIntegrationTest.java

```diff
@@ -351,4 +349,29 @@ void testGetPlagiarismCaseForStudent() throws Exception {
             assertThat(submission.getPlagiarismComparison().getSubmissionB()).as("should prepare plagiarism case response entity").isNull();
         }
     }
+
+    @Test
+    void testPlagiarismCase_getStudents() throws Exception {
```

Comment:
Thanks for your feedback, in our server tests, we frequently utilize `_` to define a sub-topic for the given test case. e.g., we have test methods named `testSavePlagiarismCaseVerdict_pointDeduction` and `testSavePlagiarismCaseVerdict_warning` in the same file. Here I named it this way since the aim of this test was testing the `PlagiarismCase::getStudents` method.